### PR TITLE
Release notes: Building encrypted image and deploy it with EAA.

### PR DESCRIPTION
**Pull request by jialez0**

### CODE REVIEW

File: releases/release_notes_09_2022.md [hunk 1]
```diff
@@ -293,5 +293,58 @@ In case the user wants to run the workload on a TDX capable hardware, using QEMU
...
+```sh
+verdictd --listen <$ip>:<$port> --mutual
+```
+
+**Note:** The communication between Verdictd and EAA KBC is based on rats-tls,
+so you need to confirm that [rats-tls](https://github.com/inclavare-containers/rats-tls) has been correctly installed in your running environment.
+
+3. Runtime Configuration
+
+`aa_kbc_params= 'eaa_kbc::<$ip>:<$port>'` should have the same ip and port with verdictd.
```
Comment (fitzthum) [inline releases/release_notes_09_2022.md:347]: This needs to be added to the agent config file, right? I'm not sure that `Runtime Configuration` is the correct heading. Could you be a little bit more specific?

**Review by fitzthum — COMMENTED** [review]
One comment but I think this is generally good.

File: releases/release_notes_09_2022.md (hunk 1 above)
Comment (jialez0) [reply releases/release_notes_09_2022.md]: OK, I have given a more explicit statement here.

**Review by sameo — APPROVED** [review]

**Comment by sameo** [comment]
@jialez0 Could you please rebase that PR so that we can merge it?

**Pull request merged**